libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -2284,7 +2284,9 @@ export async function loadDynamicValuesForParameter(
     propertiesToUpdate = { dynamicData: { status: DynamicCallStatus.SUCCEEDED }, editorOptions: { options: dynamicValues } };
   } catch (error: any) {
     const rootMessage = parseErrorMessage(error);
-    const message = error?.response?.data?.error?.message ?? rootMessage;
+    const errorData = error?.response ? error?.response?.data : error?.data;
```

Comment:
This seems like something which should go into parseErrorMessage function instead?

Reply:
The two lines should go in here so we don't have to explicitly fix this for all scenarios - libs/logic-apps-shared/src/utils/src/lib/helpers/http.ts
![image](https://github.com/user-attachments/assets/fa6fb174-5e19-431b-95f2-f7248b8a2bdb)